src/puppet-web/puppet-web.ts

```diff
@@ -336,7 +338,7 @@ export class PuppetWeb extends Puppet {
     }
   }
 
-  private async uploadMedia(mediaMessage: MediaMessage, toUserName: string): Promise<string> {
+  private async uploadMedia(mediaMessage: MediaMessage, toUserName: string): Promise<any> {
```

Comment:
The same suggestion as the above review: do not use `any`.

I think here need `interface` too.